### ISSUE TITLE
[8.0] account_banking_payment_transfer: error while making payments

### ISSUE THREAD

**Issue by eLBati** [OCA/bank-payment#369]
Steps:
 - Install ```account_banking_payment_transfer```
 - Create supplier invoice
 - Create payment order
 - Import the supplier invoice
 - Confirm
 - 'Make Payments'

Get

```
  File "parts/odoo/openerp/sql_db.py", line 247, in execute
    res = self._obj.execute(query, params)
ValueError: "syntax error at or near ")"
LINE 2:             where id in () and reconcile_id is null
                                 ^
" while evaluating
u'test_done()'
```

I will debug further.
Meanwhile, @hbrunn do you have any hints? I think this happens after https://github.com/OCA/bank-payment/commit/716519a20c61390129a7e02725e1ba80c1e36a3f

**Comment by hbrunn** [comment]
before the call to https://github.com/OCA/bank-payment/commit/716519a20c61390129a7e02725e1ba80c1e36a3f#diff-5edd6709577eb8772b7707ffeb0abc31R112, check if `self.get_transfer_move_line_ids()` returns something, return `True` otherwise.

**Comment by eLBati** [comment]
@hbrunn thanks. See https://github.com/OCA/bank-payment/pull/371

**Comment by pedrobaeza** [comment]
Fixed finally in #367
